hybrasyl/World.cs

```diff
@@ -1652,6 +1652,49 @@ where searchTerm.IsMatch(amap.Key)
                         }
                     }
                         break;
+
+                    case "/rollchar":
+                    {
+                        // /rollchar <class> <level>
+                        // Allows you to "roll a new character" of the desired Class and Level, to see his resulting HP and MP.
+
+                        string errorMessage = "Command format is: /rollchar <class> <level>";
+                        byte level = 1;
+
+                        if (args.Length != 3)
+                        {
+                            user.SendMessage(errorMessage, 0x1);
+                        }
+                        else if (!Enum.IsDefined(typeof(Class), args[1]))
+                        {
+                            user.SendMessage("Invalid class. " + errorMessage, 0x1);
+
+                        }
+                        else if (!byte.TryParse(args[2], out level) || level < 1 || level > 99)
+                        {
+                            user.SendMessage("Invalid level. " + errorMessage, 0x1);
+                        }
+                        else
+                        {
+                            // Create a fake User, and level him up to the desired level
+
+                            var testUser = new User(this, new Client());
+                            testUser.Map = new Map();

```

Comment:
Is assigning a user to a map strictly necessary here, especially since it's a fake map? I know this is just for testing, but this seems like it could have some weird side effects,